app/controllers/application_controller.rb

```diff
@@ -1,7 +1,14 @@
 class ApplicationController < ActionController::Base
-  helper_method :current_user
+  before_action :authenticate_user!, :configure_permitted_parameters, if: :devise_controller?
+  # before_action :store_user_location!, if: :storable_location?
```

Comment:
- [ ] It would be nice if you could remove all commented unused code from your codebase as that makes your codebase not clean and professional. Let's try to have a clean and professional codebase and by so doing, we remove all commented unused codes from our codebase 👍🏽